src/lib/utils/ArtworkInquiry/ArtworkInquiryStore.tsx

```diff
@@ -48,6 +49,11 @@ export const reducer = (
         ...inquiryState,
         inquiryQuestions: newSelection,
       }
+    case "setMessage":
+      return {
+        ...inquiryState,
+        message: action.payload,
```

Comment:
I think it'd be good for us to remove any extra whitespace from the payload to the user isn't just submitting with a lot of whitespace or that there's weird padding in the email that goes out to partners. 

```suggestion
        message: action.payload.trim(),
```